innovation.css

```diff
@@ -112,16 +112,17 @@
 
 /* Artifacts on display */
 .display_container {
-    background-color: #FFFFFF;
+    background-color: #FFFFFF; /* required for child div to have a nicer look using transparncy rgb background color*/
     border: 1px solid black;
     width: 195px;
     height: 165px;
     margin-bottom: 5px;
 }
 
 .display_container > div {
-    background-color: #ebd5bd;
+    /* background-color: #ebd5bd; This seems to be always overridden by the player color */
```

Comment:
I think this should be deleted let me know if my thought is correct.

Reply:
Sounds reasonable to me!

Reply:
I removed it.